yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -29,68 +33,37 @@
                     }
                 }
             </script>
-
-</body>
-</html>
+        </div>
         {% endif%}
-    </div>
-    <link rel="stylesheet" type="text/css" href="{% static 'design/detailed_design.css' %}">
-    <div class=workflow_id>
-        Workflow ID: {{ workflow.id }}
-        <br>
-        Author: {{workflow.user}}
-    </div>
-
-
-    <div class=title>
-        {{ workflow.title}}
-    </div>
+        <div class="workflow-id">
+            Workflow ID: {{ workflow.id }}
+            <br>
+            Author: {{workflow.user}}
+        </div>
 
-    <ol id="elementStyle">
-        <script type="text/javascript" src={% static "scripts/home_render_options.js" %}></script>
-            <li>
-                <div id="graph1" class = workflow-image></div>
-            </li>
-            <script type="text/javascript">
-                graph = `{{ version.yw_graph_output|safe }}`
-            </script>
-            <script type="text/javascript" src={% static "scripts/graphviz_render.js" %}></script>
-    </ol>
-
-    {% if versions %}
-        <div class="dropdown">
-                <button class="dropbtn">
-                    {% for version_ in versions %}
-                        {% if version_.id == version.id %}
-                            <text>Version #{{ forloop.counter }}  &#9662</text>
-                            <br>
-                        {% endif %}
-                    {% endfor %}
-                </button>
-                <div class="dropdown-content">
-                    {% for version_ in versions %}
-                        {% if version_.id != version.id %}
-                            <a href="{% url 'detailed_workflow' workflow.id version_.id %}">Version #{{ forloop.counter }}</a>
-                        {%  endif %}
-                    {% endfor %}
-                </div>
-            </div>
-    {% endif %}
-
-
-    <div class=outside_data>
-        <b>Description: </b>{{workflow.description}}
-        <br><br>
-       <b>Tags: </b> 
-        {% for tag in tags %}
-            <span class=tags> {{tag}}{% if not forloop.last %},{% endif %} </span>
-        {% endfor %} 
-    </div>
-    <hr />
-    <div class=run-text>
-        Runs associated with current version:
+        <div class="title">
+            {{ workflow.title}}
+        </div>
+        
+        {% include 'includeable/graph_box.html' with yw_graph_output=version.yw_graph_output %}
+        
+        <div class="detailed-dropdown-container">
+            {% include 'includeable/dropdown_selection.html' with name='Version' current=version list=versions %}
+        </div>
+        
+        <div class="outside_data">
+            <b>Description: </b>{{workflow.description}}
```

Comment:
Is this intentionally outside of the brackets?

Reply:
Yes, it was already like that and I tried to leave the current styling alone as much as I could